autoreject/tests/test_autoreject.py

```diff
@@ -195,6 +201,24 @@ def test_autoreject():
     if not np.isscalar(is_same):
         is_same = np.isscalar(is_same)
     assert_true(not is_same)
+
+    # test that transform ignores bad channels
+    picks_without_bads = mne.pick_types(
+        epochs_with_bads.info, meg='mag', eeg=True, stim=False, eog=False,
+        include=[], exclude='bads')
```

Comment:
this shouldn't be needed @mattboggess. See this: https://github.com/autoreject/autoreject/blob/master/autoreject/utils.py#L67-L68. `_handle_picks` should have taken care of the bad channel because it is called at the beginning of every function, no? If you use `picks=None`, if it does not ignore the bad channel then there is a bug. But we should make it work as before :-)

Reply:
The bads should be ignored with picks=None as well. Nothing I edited should have changed this since _handle_picks will just replace the picks before all of my changes. I use the picks here to match the clean data generated from the previous tests for comparison with and without bad channels. Do you want me to add a separate test for picks=None?